test/specs/init.js

```diff
@@ -29,6 +29,21 @@ module.exports = function (promise) {
       expect(fsm.current).to.be.equal('green');
     });
 
+    it('should accept an Object for events property', function () {
+      StateMachine.Promise = promise;
+
+      var fsm = StateMachine({
+        events: {
```

Comment:
I have state machines that model the scoring systems in different sports. In this context the events and the state names are taken from the nomenclature of the respective sport. This is a simplified example:

```
[
 {name: 'end', from: 'Round1', to: ['Break', 'Final'], condition: ... },
 {name: 'end', from: 'Break', to: 'Round2' },
]
```

Using the array format I have the flexibility to do whatever is needed to implement the rules but also present them in a familiar form to the users.